proof-systems/src/darlin/t_dlog_acc_marlin/iop/mod.rs

```diff
@@ -0,0 +1,216 @@
+//! Submodule that implements the algebraic oracle proof for t-dlog-accumulator Marlin.
```

Comment:
As the discussion we had some time ago: we can generalize everything for sure but it's not really worth it at the moment and will lead to other time spent in refactoring rather than implementing core functionalities.
So let's postpone this need of generalizing Marlin/IOP, perhaps I will open an issue about that.
It would still be nice though, as stated in other comments, to re-use some code